tests/netstats_l2/tests/01-run.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+EXPECTED_HELP = (
+    'Command              Description',
+    '---------------------------------------',
+    'reboot               Reboot the node',
+    'ifconfig             Configure network interfaces',
+    '>'
+)
+
+EXPECTED_IFCONFIG = (
+    r'ifconfig',
+    r'Iface  \d+   HWaddr: ([0-9a-f]{2}:){1,}([0-9a-f]{2}) (Channel: [11,26]  )?(Page: \d+  )?(NID: 0x[0-9a-f]{2})?',
+    r'       Source address length: \d+',
+    r'       Statistics for Layer 2',
+    r'        RX packets \d+  bytes \d+',
+    r'        TX packets \d+ \(Multicast: \d+\)  bytes \d+',
+    r'        TX succeeded \d+ errors 0'
```

Comment:
Other issue: I would not expect that `errors` is always 0, so please make that `\d+` as well.

Reply:
Done